README.md

```diff
@@ -1 +1,35 @@
-# greptime-proto
\ No newline at end of file
+# greptime-proto
+
+GreptimeDB protobuf files.
+
+## Usage
+
+### Rust
+
+```Toml
+# Add this repository as dependency to your Cargo.toml file:
+greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git" }
+```
+
+```Rust
+// To use the GreptimeDB's gRPC service:
+use greptime_proto::v1::*;
+
+// To talk to GreptimeDB Meta service:
+use greptime_proto::v1::meta::*;
+
+// To request GreptimeDB as Prometheus remote read/write:
+use greptime_proto::prometheus::remote::*;
+```
+
```

Comment:
Consider add sections for usage after code generation.

Given a greptimedb instance and its <host:4001> address, how to use generated code to call the service? As we are using Arrow Flight instead of original gRPC, a simple guide is needed to bridge the gap.